modules/node_modules/@ncigdc/utils/survivalplot.js

```diff
@@ -41,7 +41,7 @@ export const getDefaultCurve = async ({ slug, totalCases, currentFilters }: TPro
   };
 };
 
-export const getSurvivalCurves = async ({ value, field, slug, currentFilters }: TPropsMulti): Promise<Object> => {
+export const getSurvivalCurves = async ({ value, field, slug, currentFilters, shortMutationId }: TPropsMulti): Promise<Object> => {
```

Comment:
I don't think there is any need to add this logic here seeing as it's only temporary. just pass the 8 char string as `slug`